AEM/getting-started-with-automixedprecisionmkl.md

```diff
@@ -1,60 +1,63 @@
 
 ## Overview
-Mixed precision is the use of both 16-bit and 32-bit floating-point types in a model during training to make it run faster and use less memory.
+[Mixed precision](https://www.tensorflow.org/guide/mixed_precision) is the use of both 16-bit and 32-bit floating-point types in a model during training and inference to make it run faster and use less memory. Official Tensorflow* supports this feature from TensorFlow 2.9 with all Intel® optimizations enabled by default. Intel® 4th Gen Xeon processor codenamed Sapphie Rapids(SPR) is the first Intel® processor to support Advanced Matrix Extensions(AMX) instructions, which help to accelerate matrix heavy workflows such as machine learning alongside also having BFloat16 support. Previous hardwares can be used to test functionality, but SPR provides the best performance gains. 
 
-There are two options to enable BF16 mixed precision in Intel Optimized TensorFlow.
+Using one of the methods described here will convert a model written in FP32 data type to operate in BFloat16 data type. To be precise, it scans the data-flow graph corresponding to the model, and looks for nodes in the graph (also called as operators) that can operate in BFloat16 type, and inserts FP32ToBFloat16 and vice-versa Cast nodes in the graph appropriately.
+
+For example: Consider a simple neural network with a typical pattern of Conv2D -> BiasAdd -> Relu with the default TensorFlow datatype FP32. TensorFlow* data-flow graph corresponding to this model looks like below.
+
+![amp-mkl](/content/dam/develop/external/us/en/images/automixedprecisionmkl-1.png)
+
+The data-flow graph after porting the model to BFloat16 type looks like below.
+
+![amp-mkl2](/content/dam/develop/external/us/en/images/automixedprecisionmkl-2.png)
+
+Notice that 2 operators, Conv2D+BiasAdd and ReLU in the graph are automatically converted to operate in BFloat16 type. Also note that appropriate Cast nodes are inserted in the graph to convert TensorFlow* tensors from FP32 type to BFloat16 type and vice-versa.
+
+Here are the options to enable BF16 mixed precision in TensorFlow
 
 1. Keras mixed precision API
-2. AutoMixedPrecision oneDNN BFloat16 grappler pass through low level session configuration
+2. Transfer learning usecase with Tensorflow Hub SavedModel
```

Comment:
This is for transfer learning as well inference on hub or saved models right?

Reply:
yes, that's right.
Should i rephrase as "Mixed Precision for Tensorflow Hub SavedModel"? and then later mention in the description that "this works for SavedModel inference and transfer learning usecases for models from TF Hub" ?

Reply:
Yes.